arxiv-matplotlib.ipynb

```diff
@@ -204,50 +227,32 @@
    "source": [
     "### Map function across every directory\n",
     "\n",
-    "Let's scale up this work across all of the directories in our dataset\n",
+    "Let's scale up this work across all of the directories in our dataset.\n",
     "\n",
-    "Hopefully it will also be faster because the Dask workers are in the same region as the dataset itself."
+    "Hopefully it will also be faster because the cloud VMs are in the same region as the dataset itself."
    ]
   },
   {
    "cell_type": "code",
    "execution_count": null,
-   "id": "6ff8f366-4cdf-4ac3-a0c1-08545f6e07ac",
+   "id": "89a6ab9f-3451-48dc-abb2-f4d8f5e0b038",
    "metadata": {
     "tags": []
    },
    "outputs": [],
    "source": [
     "%%time\n",
     "\n",
-    "from dask.distributed import wait\n",
-    "\n",
-    "futures = client.map(extract, directories)\n",
-    "wait(futures)\n",
-    "\n",
-    "# We had one error in one file.  Let's just ignore and move on.\n",
-    "good = [future for future in futures if future.status == \"finished\"]\n",
-    "\n",
-    "lists = client.gather(good)"
+    "results = extract.map(directories[:1000])\n",
+    "lists = list(results)"
```

Comment:
Why this subsetting?

Reply:
That's a leftover from testing things out earlier. Just pushed a commit removing it